src/app/new-page/component/content-image/content-image.component.ts

```diff
@@ -57,6 +58,10 @@ export class ContentImageNewComponent implements OnChanges {
       this.imgResource =
         this.pageService.findAttachment(this.image.resource.name) || '';
     }
+
+    this.width = (this.image.width as any)?.value
+      ? `${(this.image.width as any)?.value}px`
+      : null;
```

Comment:
there are 2 types of width object that can be returned, a fixed width and a percentage width.

Reply:
```typescript
export declare namespace org.cru.godtools.shared.tool.parser.model {
    abstract class Dimension {
        protected constructor();
    }
    namespace Dimension {
        class Percent extends org.cru.godtools.shared.tool.parser.model.Dimension {
            private constructor();
            get value(): number;
            copy(value?: number): org.cru.godtools.shared.tool.parser.model.Dimension.Percent;
            toString(): string;
            hashCode(): number;
            equals(other: Nullable<any>): boolean;
        }
        class Pixels extends org.cru.godtools.shared.tool.parser.model.Dimension {
            private constructor();
            get value(): number;
            copy(value?: number): org.cru.godtools.shared.tool.parser.model.Dimension.Pixels;
            toString(): string;
            hashCode(): number;
            equals(other: Nullable<any>): boolean;
        }
    }
}
```

Reply:
`Percent` returns a float between `0` and `1`

Reply:
I created a Dimension Parser for understanding If it's a percentage or pixel.